www/stylesheets/app.css

```diff
@@ -17,6 +49,58 @@ body {
 .btn-link.btn-warning { color: #8a6d3b }
 .btn-link.btn-danger { color: #a94442 }
 
+.mission-status-icon .glyphicon {
+	vertical-align: baseline;
+	font-size: 80%;
+}
+
+.mission-stars {
+	padding-left: 10px;
+}
+.mission-stars .glyphicon {
+	padding-left: 10px;
+}
+.mission-stars .glyphicon.text-success {
+	color: #FFD800;
+}
+
+.panel-mission-status {
+	color: white;
+	text-align: center;
+	vertical-align: middle;
+}
+
+.panel-status-success .panel-mission-status { background-color: #5cb85c; }
+.panel-status-open .panel-mission-status { background-color: #d9534f; }
+.panel-status-locked .panel-mission-status { background-color: #aaa; }
+
+.panel-status-success, .panel-status-locked { opacity: 0.50 }
+.panel-status-success:hover, .panel-status-locked:hover { opacity: 1 }
```

Comment:
I'm not sure why you need some hovering. When trying, I was confused because: 1. the -open version has no hovering 2. I usually expect that hovering means that the whole area is active (clickable)